pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -143,8 +143,12 @@ def test_py_script_src_not_found(self):
             self.check_js_errors()
 
         error_msg = str(exc.value)
+        print(error_msg)
         if self.is_fake_server:
-            assert "Failed to fetch" in error_msg
+            assert (
```

Comment:
Probably late but this error message and the one below it are extremely similar.
I guess we can make the error message common and then remove the `if/else` check here.

Reply:
cc: @tedpatrick